konfera/settings.py

```diff
@@ -11,9 +11,31 @@
                             (45, _('45 min')),
                         ))
 
-"""LANDING PAGE is a composite of two keywords: <timewise>_<event>
+"""
+LANDING PAGE is a composite of two keywords: <timewise>_<event>
 <timewise> can be: latest or earliest
 <event> can be (at the moment): conference or meetup
 possible combinations: latest_conference (DEFAULT), latest_meetup, earliest_conference, earliest_meetup
 """
 LANDING_PAGE = getattr(settings, 'LANDING_PAGE', 'latest_conference')
+
+"""
+Register email notification.
+"""
+REGISTER_EMAIL_NOTIFY = getattr(settings, 'REGISTER_EMAIL_NOTIFY', False)
+REGISTER_EMAIL_BCC = getattr(settings, 'REGISTER_EMAIL_BCC', [])
+REGISTER_EMAIL = getattr(settings, 'REGISTER_EMAIL',
+                         'Dear {first_name} {last_name},\n\n' \
```

Comment:
I'd rather have this in a separate file (`register/email.txt` ?) than in the settings.